core/extension/gdextension_interface.h

```diff
@@ -198,6 +198,7 @@ typedef struct {
 
 typedef void (*GDExtensionVariantFromTypeConstructorFunc)(GDExtensionUninitializedVariantPtr, GDExtensionTypePtr);
 typedef void (*GDExtensionTypeFromVariantConstructorFunc)(GDExtensionUninitializedTypePtr, GDExtensionVariantPtr);
+typedef void *(*GDExtensionInterfaceVariantGetInternalPtrFunc)(GDExtensionVariantPtr);
```

Comment:
We use the `GDExtensionInterface*` prefix for the function pointers we load from Godot; for others we don't include the word `Interface`. Maybe this could be:

```suggestion
typedef void *(*GDExtensionVariantGetInternalPtrFunc)(GDExtensionVariantPtr);
```